### PR TITLE
Modify disk space management page

### PR DESCRIPTION
**Pull request by gjbex**
Based on your feedback and cross-checking with Stefan on CUE I modified the page on disk space management accordingly.

### CODE REVIEW

File: source/access/managing_disk_usage.rst [hunk 1]
```diff
@@ -5,98 +5,47 @@ How much disk space am I using?
...
-   about the quota on your ``$VSC_HOME`` and ``$VSC_DATA``
-   directories and will show you  the quota for your ``$VSC_SCRATCH``
-   directory on that system.
+On most systems, ``myquota`` will show you for the ``$VSC_HOME``,
+``$VSC_DATA`` and ``$VSC_SCRATCH`` file systems either which
+percentage of the available disk space you are using, or the
+absolute amount. Users from Ghent university should check their disk usage
+using the `web application <https://account.vscentrum.be/>`_.
 
+If quota have been set on the number of files you can create on a file
```
Comment (itkovian) [inline source/access/managing_disk_usage.rst]: Technically, that would be inodes on GPFS, no? Which is not exactly the same.

File: source/leuven/mmlsquota.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.. _mmlsquota:
+
+Total disk space on GPFS file systems
+=====================================
+
+On some parallel file systems, notably ``$VSC_SCRATCH`` on KU Leuven
+infrastructure (Tier-1 and Tier-2), ``mmlsquota`` should be used to
+display the total disk space you use and the limits you have.
+
+These file system is a parallel file system, i.e., IBM Spectrum Scale (formerly
```
Comment (itkovian) [inline source/leuven/mmlsquota.rst]: Either `This file system` or `These file systems`.

File: source/leuven/mmlsquota.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+GPFS)::
+
+      $ mmlsquota  --block-size auto  vol_ddn2:leuven_scratch
+                               Block Limits                                                   |     File Limits
+      Filesystem Fileset        type             KB      quota      limit   in_doubt    grace |    files   quota    limit in_doubt    grace  Remarks
+      vol_ddn2   leuven_scratch  USR        1444864 1970061312 2073755648          0     none |    16494       0        0        0     none
+
+The ``--block-size auto`` option respectively ensure that sizes are
```
Comment (itkovian) [inline source/leuven/mmlsquota.rst]: `ensures`

Comment (itkovian) [reply source/leuven/mmlsquota.rst]: What is the `respectively` doing there?

File: source/leuven/mmlsquota.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+reported in a human-readable format, rather than expressed in blocks.
+
+The optional volume ``vol_ddn2:leuven`` is the fileset you want information
```
Comment (itkovian) [inline source/leuven/mmlsquota.rst]: `In the above example, ...`

File: source/leuven/mmlsquota.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+on.  You can leave out the fileset, but in that case ``mmlsquota`` will
+display information on all filesets, most of which you don't have access
```
Comment (itkovian) [inline source/leuven/mmlsquota.rst]: Wait, what? So you can glean information on stuff you are not entitled to? How nice.

File: source/access/managing_disk_usage.rst [hunk 6]
```diff
@@ -5,98 +5,47 @@ How much disk space am I using?
...
-
-.. note::
-
-   Using these commands on another cluster than the one
-   in your home institution, will fail to return information
-   about the quota on your ``$VSC_HOME`` and ``$VSC_DATA``
-   directories and will show you  the quota for your ``$VSC_SCRATCH``
-   directory on that system.
+On most systems, ``myquota`` will show you for the ``$VSC_HOME``,
+``$VSC_DATA`` and ``$VSC_SCRATCH`` file systems either which
```
Comment (itkovian) [inline source/access/managing_disk_usage.rst]: `either the percentage ... `

File: source/leuven/mmlsquota.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
+.. _mmlsquota:
+
+Total disk space on GPFS file systems
+=====================================
+
+On some parallel file systems, notably ``$VSC_SCRATCH`` on KU Leuven
+infrastructure (Tier-1 and Tier-2), ``mmlsquota`` should be used to
+display the total disk space you use and the limits you have.
```
Comment (smoors) [inline source/leuven/mmlsquota.rst]: @gjbex why `myquota` cannot be used for for $VSC_SCRATCH on KU Leuven infra as well?

File: source/access/managing_disk_usage.rst (hunk 1 above)
Comment (gjbex) [reply source/access/managing_disk_usage.rst]: I think explaining the difference between inodes and actual file counts would only muddle matters.  The point is that people have a quick way to see they're using too much whatever.

I'd suggest to use "files" as proxy.

File: source/leuven/mmlsquota.rst (hunk 7 above)
Comment (smoors) [reply source/leuven/mmlsquota.rst]: replacing 'should' with 'can' is indeed better in that case.

**Comment by gjbex** [comment]
All feedback taken into account (except the remark on inodes).

**Comment by gjbex** [comment]
Many thanks to all.

**Pull request merged**